examples/maps/maps.bal

```diff
@@ -28,19 +32,29 @@ public function main() {
     // Prints the number of mappings in the `map`.
     io:println(addrMap.length());
 
-    // Mappings can be removed using the `remove()` method.
-    boolean isRemoved = addrMap.remove("postalCode");
+    // Mappings can be removed using the `.remove()` method.
+    any isRemoved = addrMap.remove("postalCode");
     io:println(addrMap);
 
+    // Maps support functional iterations.
+    addrMap.forEach(function (any value) {
```

Comment:
This doesn't have to be `any`. This can simply be the constraint type of the map